src/ItemValue.ts

```diff
@@ -132,7 +132,7 @@ export class ItemValue implements IItemValue {
                 this.update();
             } else {
                 // Otherwise save the new version to memory
-                this.updateLocalStorage();
+                this.updateLocalStorage(true);
```

Comment:
I believe that's the intended purpose. If you create a new `ItemsHoldr` with no autosave and defaults defined in settings, it shouldn't write them to localStorage before manual saving is requested.